controllers/module_reconciler.go

```diff
@@ -108,6 +108,7 @@ func NewModuleReconciler(
 //+kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=rolebindings,verbs=create;delete;get;list;patch;watch
 //+kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=clusterroles,verbs=bind,resourceNames=module-loader;device-plugin
 //+kubebuilder:rbac:groups="build.openshift.io",resources=builds,verbs=get;list;create;delete;watch;patch
+//+kubebuilder:rbac:groups="batch",resources=jobs,verbs=create;list;watch;delete
```

Comment:
Please run `make manifests bundle` and add the generated files to the PR.

Reply:
FYI @enriquebelarte.

Seems like `make bundle` is also needed for making sure we are pushing generated files.

Reply:
`make manifests bundle` runs both targets 🙂

Reply:
Yes. @enriquebelarte and I were talking about a job to make sure generated files aren't forgotten from PRs and we only talked about `make generate manifests` so I wan't to mention that we also need `make bundle` in the job.

This was intended for Enrique :)

Reply:
OK, I'll create a new PR for that.

Reply:
ok I *think* I have this right, turns out if you rebase onto main first running `make manifests bundle` generates a load of other changes that are not related to this PR, so I have un-rebased and (hopefully!) I just have my changes, but I think that means we need another PR that generates the other changes (related to moving form dockerfiles to config maps iirc)

Reply:
Indeed. We already have a job u/s to make sure generated files are not forgotten. Once it is pulled m/s by the new sync tool we will have that validation as well m/s.